cmd/root.go

```diff
@@ -45,8 +45,10 @@ var RootCmd = &cobra.Command{
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
 	if err := RootCmd.Execute(); err != nil {
-		fmt.Println(err)
-		os.Exit(1)
+		if (err != nil) && debug {
+			fmt.Printf("%+v\n", err)
+		}
```

Comment:
log fatal will print the error

Reply:
but nit the stack - hence inside the `if debug` block... but another check on err is unnecessary, removed